akvo/rsr/spa/app/modules/editor/project-init-handler.js

```diff
@@ -11,10 +11,12 @@ const insertRouteParams = (route, params) => {
   })
   return route
 }
-let prevParams
+
 let sectionIndexPipeline = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11]
 
 const ProjectInitHandler = connect(({editorRdr}) => ({ editorRdr }), actions)(React.memo(({ match: {params}, ...props}) => {
+  const [nextSectionIndex, setNextSectionIndex] = useState(0)
```

Comment:
I know nothing of react, so maybe this is OK, but state `0`? Will it always be `0` and how do we know it will stay `0`?

Reply:
Thanks @MichaelAkvo , so Actually I'm trying to move from uncontrolled state to controlled state.
```useState``` is helps us to controlled state and set initial value being updated along with React live cycle.

So for this case, I'm trying to set initial value for nextSectionIndex is 0. and this state will be use for calling index of Array and it's will be updated via ```setNextSectionIndex``` function.

Reply:
I see. Thanks for explaining